libraries/botbuilder/tests/teamsActivityHandler.test.js

```diff
@@ -61,6 +61,226 @@ describe('TeamsActivityHandler', () => {
         });
     });
 
+    describe('onInvokeActivity()', () => {
+        class InvokeActivity extends TeamsActivityHandler{
+            constructor(){
+                super();
+            }
+
+            handleTeamsO365ConnectorCardAction(context, query) {
+                return { status: 501 };
+            }
+
+            handleTeamsAppBasedLinkQuery(context, query) {
+                return { status: 501 };
+            }
+
+            handleTeamsMessagingExtensionQuery(context, query) {
+                return { status: 501 };
+            }
+
+            handleTeamsMessagingExtensionSelectItem(context, query) {
+                return { status: 501 };
+            }
+
+            handleTeamsMessagingExtensionFetchTask(context, action) {
+                return { status: 501 };
+            }
+
+            handleTeamsMessagingExtensionConfigurationQuerySettingUrl(context, action) {
+                return { status: 501 };
+            }
+
+            handleTeamsMessagingExtensionConfigurationSetting(context, settings) {
+                return { status: 501 };

```

Comment:
If all you're trying to do is not trigger the thrown error from the default implementation, then you don't need to return anything.

Returning `{ status: 501 };` is confusing.